webrtc.html

```diff
@@ -5347,7 +5458,28 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
       <code><a>RTCRtpTransceiver</a></code> is defined by the kind of the
       associated <code><a>RTCRtpReceiver</a></code>'s
       <code><a>MediaStreamTrack</a></code> object.</p>
-      <div>
+      <p>To <dfn>create an RTCRtpTransceiver</dfn> with an <code>
+      <a>RTCRtpReceiver</a></code> object, <var>receiver</var>, an <code>
+      <a>RTCRtpSender</a></code> object, <var>sender</var>, run the following
+      steps:</p>
+      <ol>
+        <li>
+          <p>Let <var>transceiver</var> be a new <code>
+          <a>RTCRtpTransceiver</a></code> object.</p>
+        </li>
+        <li>
+          <p>Let <var>transceiver.sender</var> be <var>sender</var>.</p>
```

Comment:
I'm using "set" for everything that could be considered to have a value before to be consistent.